d2l/train.py

```diff
@@ -43,8 +52,7 @@ def train_ch3(net, train_iter, test_iter, criterion, num_epochs, batch_size, lr=
             train_acc_sum += torch.sum((torch.argmax(y_hat, dim=1).type(torch.FloatTensor) == y).detach()).float()
             n += list(y.size())[0]
         test_acc = evaluate_accuracy(test_iter, net)
-        print('epoch %d, loss %.4f, train acc %.3f, test acc %.3f'
-              % (epoch + 1, train_l_sum / n, train_acc_sum / n, test_acc))
+        print('epoch %d, loss %.4f, train acc %.3f, test acc %.3f'% (epoch + 1, train_l_sum / n, train_acc_sum / n, test_acc))
```

Comment:
Don't make this a single line. See PEP-8 formatting style. 80 tokens at max in a line.